app/src/main/java/com/boolder/boolder/view/detail/ProblemBSFragment.kt

```diff
@@ -201,6 +215,24 @@ class ProblemBSFragment(private val listener: BottomSheetListener) : BottomSheet
             }
         }
 
+        binding.tick.setOnClickListener{
+            val tickRepository: TickRepository = get()
```

Comment:
Having the repository set as a class attribute will avoid the need to get it from the dependency injection again

Reply:
See https://github.com/boolder-org/boolder-android/pull/33#discussion_r1303384910